test/auro-checkbox.test.js

```diff
@@ -15,6 +15,8 @@ describe('auro-checkbox-group', () => {
       <auro-checkbox-group
         horizontal
         required
+        aria-invalid
+        aria-required
```

Comment:
Why are these added here? The test should be that the code adds them based on validityState and the presence of the `required` attribute.